croud/rest.py

```diff
@@ -97,7 +97,12 @@ def send(
             data = body["data"] if "data" in body else body
             return data, None
 
-    def print(self, success_message: str = None, keys: List[str] = None):
+    def print(
+        self,
+        success_message: str = None,
+        keys: List[str] = None,
+        level: str = Level.SUCCESS.value,
```

Comment:
Let's make `level` a `Level` type, not a string. You can then pass the enum directly and not the string itself:
```suggestion
        level: Level = Level.SUCCESS,
```